core/src/main/java/io/questdb/cairo/TableConverter.java

```diff
@@ -168,6 +169,8 @@ private static boolean readWalEnabled(LPSZ path, FilesFacade ff) {
             final byte walType = ff.readNonNegativeByte(fd, 0);
             switch (walType) {
                 case TABLE_TYPE_WAL:
+                    // fall through
+                case TABLE_TYPE_MAT:
```

Comment:
Do we allow converting MAT views into non-wal tables? Is it tested?

Reply:
No, all alter table operations are banned for mat views. I've added more test cases in adefb14d217421024318d4dde4960a9b8ae4ccd6

Reply:
btw, this is not longer the case.
Enabled resume/suspend wal for mat views, but all other alters are rejected, that is correct.

Reply:
Yes, thanks to Imre, `ALTER MATERIALIZED VIEW xyz RESUME WAL` is now supported.